integration-tests/src/test/resources/field_level_security_mode/readonlyrest.yml

```diff
@@ -0,0 +1,32 @@
+readonlyrest:
+  ssl:
+    enable: true
+    keystore_file: "keystore.jks"
+    keystore_pass: readonlyrest
+    key_pass: readonlyrest
+
+  access_control_rules:
+
+    # ES containter initializer need this rule to configure ES instance after startup
+    - name: "CONTAINER ADMIN"
+      auth_key: admin:container
+      verbosity: error
+
+    - name: fls_mode_default
+      auth_key: "user1:pass"
+      fields: ["_source","allowedField"]
+
+    - name: fls_mode_hybrid_explicit
+      auth_key: "user2:pass"
+      fields: ["_source","allowedField"]
+      fields_mode: "hybrid"
```

Comment:
we should avoid this approach in new rules or during extending existing rules. Please take a look at `proxy_auth` rule decoder and do it in the same way - by defining simple settings decoder and extended settings decoder.

Extended version can be defined like in the following example:
```
fields: 
  list: ["_source", "allowedField"]
  mode: "hybrid"
```